python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1407,3 +1477,496 @@ def _compute_model_params(self, wcsf):
             modelParams[key] = np.array(value)
 
         return modelParams
+
+
+class GbdesGlobalAstrometricFitConnections(
```

Comment:
Good question--this is "global" in the sense that it is not tract-based, which is how "global" is sometimes used in the pipeline language, but you are correct that it can't actually be arbitrarily run on all data. I think I'll take this to #dm-naming-things unless you have any ideas to suggest.